.travis.yml

```diff
@@ -117,6 +117,22 @@ jobs:
           tags: false
           repo: triplea-game/triplea
           branch: master
+    - stage: deploy prerelease
+      if: (branch = master) and (repo = 'triplea-game/triplea') and (type != 'pull_request')
+      install:
+        - sudo apt update
+        - sudo apt -y install sshpass
+        - sudo pip install ansible
```

Comment:
Depending on the way travis is setup here, you might not even need to use sudo for this command.

Regardless there has to be a way to install python packages on travis without sudo

Reply:
Was able to remove the `sudo` on pip: 8eaa2fae2